tests/integration/conftest.py

```diff
@@ -516,3 +524,30 @@ async def juju_image_id_fixture(
     )
     assert image, "Juju image not found"
     return image.id
+
+
+@pytest_asyncio.fixture(scope="module", name="microk8s_image_id")
+async def microk8s_image_id_fixture(
+    openstack_connection: Connection,
+    dispatch_time: datetime,
+    image_configs: ImageConfigs,
+    app: Application,
+):
+    """The Juju bootstrapped image expected from builder application."""
+    arch = _get_supported_arch()
+    image: Image | None = await wait_for(
+        functools.partial(
+            image_created_from_dispatch,
+            image_name=(
+                f"{app.name}-{image_configs.bases[0]}-{arch.value}-juju-"
+                f"{image_configs.juju_channels[0].replace('/','-')}-"
+                f"mk8s-{image_configs.microk8s_channels[0].replace('/','-')}"
+            ),
+            connection=openstack_connection,
+            dispatch_time=dispatch_time,
+        ),
+        timeout=60 * 30,
```

Comment:
nit: Such timeouts are easier to find/adjust if they are placed as constants at the top of the module.

Reply:
May I apply this as a follow up since the tests are really hard to pass due to the rate limiting and infra issues atm? :D